pro/src/pages/AdageIframe/app/components/AdageDiscovery/OfferCard/OfferCard.tsx

```diff
@@ -14,12 +14,14 @@ import styles from './OfferCard.module.scss'
 
 export interface CardComponentProps {
   offer: CollectiveOfferTemplateResponseModel
-  handlePlaylistElementTracking: () => void
+  handleTracking: () => void
+  to?: string
 }
 
 const OfferCardComponent = ({
   offer,
-  handlePlaylistElementTracking,
+  handleTracking,
+  to = '/decouverte',
```

Comment:
Détail de naming : le `to` me laisse penser qu'on va **vers** la page découverte, mais on va plutôt vers la page d'offre **depuis** la page découverte c'est ça ? Est-ce que ça aurait plus de sens de l'appeler `from` ?

Reply:
de base le composant c'était que depuis la page découverte on soit redirigé vers une page offre mais maintenant on ajoute la redirection depuis la page recherche à une page offre